docs/.vuepress/theme/components/NavLinks.vue

```diff
@@ -35,35 +135,222 @@ export default {
     }
   },
   methods: {
-    isActive
+    isActive,
+    handleSearchToggle(value) {
+      this.isSearchVisible = value
+    },
+    handleFocusToggle(value) {
```

Comment:
Similar to `handleNavToggle`, I think this could be simplified to just toggle a boolean without passing a value.